src/Listeners/UpdateUnusedAssetsCacheListener.php

```diff
@@ -13,21 +11,13 @@
 use Statamic\Events\GlobalSetSaved;
 use Statamic\Events\TermDeleted;
 use Statamic\Events\TermSaved;
-use Teamnovu\StatamicUnusedAssets\UnusedAssets;
+use Teamnovu\StatamicUnusedAssets\Jobs\UpdateUnusedAssetsCacheJob;
 
-class UpdateUnusedAssetsCacheListener implements ShouldQueue
+class UpdateUnusedAssetsCacheListener
 {
-    use Queueable;
-
-    public function __construct(
-        protected UnusedAssets $unusedAssets,
-    ) {
-    }
-
     public function handle($event)
     {
-        $this->unusedAssets->clearCache();
-        $this->unusedAssets->preloadCache();
+        UpdateUnusedAssetsCacheJob::dispatch();
     }
 
     /**
```

Comment:
We ä User saved/deleted wird, muesch ds äuä o machä. Ä User cha ja o Assets ha.

Reply:
Voll das söt me no mache